bazel/emscripten_toolchain/emscripten_config

```diff
@@ -9,7 +9,7 @@ FROZEN_CACHE = True
 
 system = platform.system()
 
-machine = "arm64" if platform.machine() == "arm64" else "amd64"
+machine = "arm64" if (platform.machine() == "arm64" or platform.machine() == "aarch64") else "amd64"
```

Comment:
It seems like mac/linux python report this differently:
```
$ python3 # on a arm mac
Python 3.10.8 (main, Oct 13 2022, 09:48:40) [Clang 14.0.0 (clang-1400.0.29.102)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import platform; platform.machine()
'arm64'
$ docker run -it --platform=linux/arm64 python
Python 3.11.1 (main, Dec 21 2022, 09:36:17) [GCC 10.2.1 20210110] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import platform; platform.machine()
'aarch64'
```

Without this, during linking I'd be hit with a:
```
emcc: warning: cannot check node version: [Errno 2] No such file or directory: '/root/.cache/bazel/.../external/nodejs_linux_amd64/bin/node' [-Wversion-check]
```

Reply:
I see now this may not be necessary after https://github.com/emscripten-core/emsdk/pull/1151